src/CMakeLists.txt

```diff
@@ -544,7 +544,7 @@ target_link_libraries(
         ZIP::ZIP
         ZLIB::ZLIB
 # PLACEMARKER: sample benchmarking code
-#       nanobench
+        # nanobench
```

Comment:
Cosmetic, I know but leaving it in the first column does make it easier to pick out:
```suggestion
#         nanobench
```